src/services/azureFunctionsService.ts

```diff
@@ -68,16 +69,34 @@ export class AzureFunctionsService implements mssql.IAzureFunctionsService {
 			return;
 		}
 		let projectFile = await azureFunctionUtils.getAzureFunctionProject();
+		let newHostProjectFile: azureFunctionUtils.IFileFunctionObject;
+		let hostFile: string;
+
 		if (!projectFile) {
 			let projectCreate = await vscode.window.showErrorMessage(LocalizedConstants.azureFunctionsProjectMustBeOpened,
 				LocalizedConstants.createProject, LocalizedConstants.learnMore);
 			if (projectCreate === LocalizedConstants.learnMore) {
 				vscode.commands.executeCommand('vscode.open', vscode.Uri.parse(constants.sqlBindingsDoc));
+				return;
 			} else if (projectCreate === LocalizedConstants.createProject) {
-				// start the create azure function project flow
-				await azureFunctionApi.createFunction({});
+				try {
+					// start the create azure function project flow
+					// create a watcher to see if the host file is created after the flow is complete
+					newHostProjectFile = await azureFunctionUtils.waitForNewHostFile();
+					await azureFunctionApi.createFunction({});
```

Comment:
So this is going to start the create project flow from the start, right? That could potentially be a problem, as we have some specific requirements such as not supporting the .NET 5 runtime currently. Are you planning on following up to ensure that this create experience isn't going to lead users down the wrong path?

Reply:
Is there documentation of what runtimes / sdks are supported in regards to SQL binding?

Reply:
I thought we did but I'm not seeing that specifically listed anywhere. https://docs.microsoft.com/en-us/azure/azure-functions/functions-bindings-azure-sql just mentions that we only support AF v2

@dzsquared Do you know if we list the requirements anywhere?